kodilla-stream/src/test/java/com/kodilla/stream/homework/ForumStatsTest.java

```diff
@@ -0,0 +1,44 @@
+package com.kodilla.stream.homework;
+
+import com.kodilla.stream.User;
+import com.kodilla.stream.UsersRepository;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class ForumStatsTest {
+
```

Comment:
gdzie warunki brzegowe? Gdyby byly, to bys zobaczyl, ze twoje metody nie dzialaja